discordClient.js

```diff
@@ -16,4 +14,21 @@ discordClient.on('ready', function (evt) {
     logger.info(discordClient.username + ' - (' + discordClient.id + ')');
 });
 
+var randomFunnyMessages = [
+    "I am back!",
+    "EROMBO",
+    "Moi Moi!",
+    "Whats up lads!",
```

Comment:
nit: I know it's just guys so far, but maybe "What's up folks!"? or imitate Gohan, "Hello Friend"

Reply:
Haha. I like that
I will make sure to include those